packages/react-ui/src/components/ui/form.tsx

```diff
@@ -158,7 +158,7 @@ const FormMessage = React.forwardRef<
     <p
       ref={ref}
       id={formMessageId}
-      className={cn('text-sm font-medium text-destructive', className)}
+      className={cn(formMessageClass, className)}
```

Comment:
nit: It's better to keep it inline to close where your need it, the variable is also used once, It's better inline.

Because imagine someone reading it, they will have to scroll up and check the variable deceleration

Reply:
the variable is exported, it is not intended to be used once, we have at least two places where we don't use the FormMessage class but still need validation, that's where the variable is used, so error messages don't look different in multiple places.